tests/neg/t5702-neg-bad-and-wild.scala

```diff
@@ -20,7 +20,7 @@ object Test {
 // good syntax, bad semantics, detected by typer
 //gowild.scala:14: error: star patterns must correspond with varargs parameters
     val K(x @ _*) = k
-    val K(ns @ _*, x) = k // error: pattern expected
+    val K(ns @ _*, xx) = k // error: pattern expected
```

Comment:
This is an interesting case. `x` started showing and error about being duplicate, which is true. I changed the condition to test the actual issue.

I wonder if I should add another case for the duplicate name?

Reply:
I added a test, the error seems legit, though I have no idea why it popped up now.